<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rtu_para</class>
 <widget class="QDialog" name="rtu_para">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">RTU基本参数录入</string>
  </property>
  <widget class="QComboBox" name="cb_rtu_station">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>变电站名称:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rtu_producer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>深圳市科陆电子有限公司</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>制造厂家:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>装置安装位置:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_position">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>521</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>深圳市南山区科技园北区酷派信息港附近</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>线路名称:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rtu_number">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>123</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>出厂编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>装置(RTU)型号:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rtu_type">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>164</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>等级:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rtu_grade">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>164</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PT二次电压(V):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_twice_v">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>205</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>57.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>220</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>380</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CT二次电流(A):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_twice_a">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>205</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PT变比:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CT变比:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>接线方式:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rtu_lineway">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>286</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯协议:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rtu_protocl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>286</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>直流电压输入:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dc_v">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>325</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>直流电压输入:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dc_A">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>325</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>检验证号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>370</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字第:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>环境温度:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_temper">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>403</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>444</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>检定员1:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_checker1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>448</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>相对湿度:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_humdity">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>403</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>444</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>检定员2:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_checker2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>448</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>改变量检定方案:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_variable_check">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>494</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_error_check">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>496</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>误差检定方案:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>记录号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tel_para">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>549</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步:遥控参数</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_rtu_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_uid_number">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>364</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_record_page">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_check_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>361</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_PT_Ratio">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>246</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_CT_Ratio">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>246</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>480</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pb_add">
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_save">
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_search">
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete">
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_previous">
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_next">
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
